### PR TITLE
Chatrooms with functions

### PR DESCRIPTION
**Pull request by Protoox**

### CODE REVIEW

File: src/main/java/com/ateam/Server.java [hunk 1]
```diff
@@ -46,10 +46,12 @@ public Server() {
         }
 
         this.userAuth = new UserAuth(db);
+        ChatsRooms.put("General", clients);
```
Comment (TSAsmilex) [inline src/main/java/com/ateam/Server.java]: "General" -> "general"

File: src/main/java/com/ateam/Server.java [hunk 2]
```diff
@@ -116,4 +117,64 @@ public void addNewClientHandler(Socket socket, UserAuth ua) throws ClientHandler
...
+        
+    }
+    
+    public void listRoom(ClientHandler client){
+        String chatslist="";
+        for (Map.Entry<String, ArrayList<ClientHandler>> chatroom : ChatsRooms.entrySet()) {
+            String key = chatroom.getKey();
+            ArrayList value = chatroom.getValue();
+            
+         chatslist += key +"[" + value.size()+"]\n";
```
Comment (TSAsmilex) [inline src/main/java/com/ateam/Server.java]: El formato del mensaje debería ser:

```
Salas disponibles: nombre_sala(4) otra_sala(5) ultima_sala(1)
```

**Comment by Protoox** [comment]
Done!

**Comment by TSAsmilex** [comment]
Methods for the new class ChatRoom:

- `size()`: number of users connected in the room
- `getRoomName()`
- `getUsers()`: get the users which are currently connected.
- `remove(Client)` boolean.
- `contains(Client)` boolean.
- `add(Client)`

**Pull request merged**